### PR TITLE
Automatic updates

### PR DESCRIPTION
**Pull request by bastianeicher**
These commits were created by [this AppVeyor CI job](https://ci.appveyor.com/project/0install/repo-roscidus-com/build/job/he7e3f6tkvd5yxmt).

@talex5 I was wondering whether it would be better to squash these commits to keep the Git history less noisy or to preserve them to show all the individual XML snippets that were fed into 0repo. What do you prefer?

### CODE REVIEW

**Comment by talex5** [comment]
Multiple commits is fine. Not sure about the blank email addresses, though. I wonder if that might cause problems in the future for some tools that try to parse them.

**Comment by bastianeicher** [comment]
We'd have to change this: https://github.com/0install/0repo/blob/master/repo/scm.py#L42
Not sure where to get the email address from though, if there no GnuPG key is specified.
I do believe that a Git commit without an email address is valid in principle.

For now I've replaced the commits with ones created by me.

**Comment by bastianeicher** [comment]
Added some additional updates.

File: utils/curl.xml [hunk 1]
```diff
@@ -283,13 +283,13 @@
       <manifest-digest sha256new="WPBA7YQ2UXZ4L3X4S6WGEYHNOGLDB6AIR2UJYWG2Q2V2V2JBGXQQ"/>
       <archive extract="curl-7.63.0-win32-mingw" href="https://bintray.com/vszakats/generic/download_file?file_path=curl-7.63.0-win32-mingw.tar.xz" size="2049216" type="application/x-xz-compressed-tar"/>
     </implementation>
-    <implementation arch="Windows-x86_64" id="sha1new=1ed19054d4ac2157074091abdc78825379dd818b" released="2019-02-06" stability="stable" version="7.64.0">
```
Comment (talex5) [inline utils/curl.xml]: Should the automatic-updater be changing existing implementations like this?

Comment (bastianeicher) [reply utils/curl.xml]: I made this change manually after I noticed that the upstream site had changed the content of the archive. Unfortunately they did this without changing the URL or version number. I'm assuming they fixed some packaging issue without rebuilding the binaries.

**Pull request merged**